### PR TITLE
[CHORE]migrate codes to kotlin

### PR DESCRIPTION
**Pull request by laiyin-xue**
for issue #2[](https://github.com/ekino/gradle-docker-plugin/issues/2)

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -33,14 +34,21 @@ val dockerComposePluginVersion = "0.9.4"
 val junitVersion = "5.5.1"
 
 dependencies {
-  implementation(localGroovy())
+  implementation(kotlin("stdlib"))
```
Comment (clemstoquart) [inline build.gradle.kts]: `stdlib-jdk8` plz.

File: build.gradle.kts [hunk 2]
```diff
@@ -33,14 +34,21 @@ val dockerComposePluginVersion = "0.9.4"
...
 
   testImplementation(gradleTestKit())
   testImplementation("org.junit.jupiter:junit-jupiter:$junitVersion")
 }
 
 tasks {
+  withType<KotlinCompile> {
+    kotlinOptions {
+      freeCompilerArgs = listOf("-Xjsr305=strict")
+      jvmTarget = JavaVersion.VERSION_11.toString()
```
Comment (clemstoquart) [inline build.gradle.kts]: I think we should keep compatibility with JDK 8 users.

File: build.gradle.kts (hunk 1 above)
Comment (laiyin-xue) [reply build.gradle.kts]: done

File: build.gradle.kts (hunk 2 above)
Comment (laiyin-xue) [reply build.gradle.kts]: done

**Comment by clemstoquart** [comment]
Thanks @laiyin-xue

**Pull request merged**